router-tests/testenv/testenv.go

```diff
@@ -628,9 +628,9 @@ func configureRouter(listenerAddr string, testConfig *Config, routerConfig *node
 			EnablePersistedOperationsCacheResponseHeader: true,
 			EnableNormalizationCacheResponseHeader:       true,
 		},
-		EpollKqueuePollTimeout:         300 * time.Millisecond,
-		EpollKqueueConnBufferSize:      1,
-		WebSocketReadTimeout:           time.Millisecond * 100,
+		WebsocketClientPollTimeout:     300 * time.Millisecond,
+		WebsocketClientConnBufferSize:  1,
+		WebsocketClientReadTimeout:     time.Millisecond * 100,
```

Comment:
Why `time * constant` and then `constant * time`? I think it's best to pick a single style and stick to it. WDYT?

Reply:
I didn't touch this. Those are style issue and should be ensured by a linter. We don't have such rules and therefore personal preference.

Reply:
It doesn't matter which is picked, but it's very odd to have two contrasting styles almost directly juxtaposed.

Reply:
Addressed, but IMO we should not focus on this in a PR.